azure/template.json

```diff
@@ -32,6 +32,12 @@
         "serviceName": {
             "type": "string"
         },
+        "sharedApimName": {
```

Comment:
```suggestion
        "sharedApimName": {
```
Can this match directly with the variable name in variable group so no extra variable is needed in pipeline variables

Reply:
Why do they need to match?  In das-employer-incentives-web I've done -sharedApimName "$(SharedApimServiceName)"

Reply:
If we can reduce the number of pipeline variables we should.

I think das-employer-incentives-web should change as well.

We follow that pattern for other variable group variables, eg. SharedManagementResourceGroup/ / SharedEnvResourceGroup

Reply:
I'ved altered the param name